tachyon/c/zk/plonk/halo2/bn254_gwc_prover_unittest.cc

```diff
@@ -126,7 +131,7 @@ TEST_P(GWCProverTest, SetRng) {
   auto cpp_generator =
       std::make_unique<RandomFieldGenerator<math::bn254::Fr>>(cpp_rng.get());
 
-  EXPECT_EQ(reinterpret_cast<Prover<PCS>*>(prover_)->blinder().Generate(),
+  EXPECT_EQ((reinterpret_cast<Prover<PCS, LS>*>(prover_)->blinder().Generate()),
```

Comment:
```suggestion
  EXPECT_EQ(reinterpret_cast<Prover<PCS, LS>*>(prover_)->blinder().Generate(),
```

Reply:
I did this on purpose, because the compiler takes `,` between `<PCS, LS>` as another argument.
So when I remove those parenthesizes, compiler takes as three arguments and I get this error:
```
error: macro "EXPECT_EQ" passed 3 arguments, but takes just 2
```
I think it's weird...